packages/analytics/analytics-metric-provider/src/components/MetricsProvider.vue

```diff
@@ -85,17 +88,34 @@ const tz = computed(() => {
   return (new Intl.DateTimeFormat()).resolvedOptions().timeZone
 })
 
+const skuFeatureFlag = computed<boolean>(() => {
+  // The feature flag client is guaranteed to be initialized by the time the code gets to this place.
+  return evaluateFeatureFlagFn('MA-2527-analytics-sku-config-endpoint', false)
+})
+
+const resolvedDatasource = computed<QueryDatasource>(() => {
+  if (props.datasource) {
+    return props.datasource
+  }
+
+  return skuFeatureFlag.value ? 'basic' : 'advanced'
```

Comment:
just commenting out loud: 

If datasource is set, use that datasource. 

if it's not set, if the skuFlag is off, default to advanced. else true -> roll basic.

:+1: